SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTestRequireHandler.cpp

```diff
@@ -378,19 +376,9 @@ void SpatialFunctionalTestRequireHandler::LogAndClearStepRequires()
 
 		for (const auto& Require : RequiresOrdered)
 		{
-			FString Msg;
-			if (Require.bPassed)
-			{
-				Msg = FString::Printf(TEXT("%s [Passed] %s : \"%s\""), *WorkerName, *Require.Msg, *Require.StatusMsg);
-				UE_VLOG(nullptr, LogSpatialGDKFunctionalTests, Display, TEXT("%s"), *Msg);
-				UE_LOG(LogSpatialGDKFunctionalTests, Display, TEXT("%s"), *Msg);
-			}
-			else
-			{
-				Msg = FString::Printf(TEXT("%s [Failed] %s : %s"), *WorkerName, *Require.Msg, *Require.StatusMsg);
-				UE_VLOG(nullptr, LogSpatialGDKFunctionalTests, Error, TEXT("%s"), *Msg);
-				UE_LOG(LogSpatialGDKFunctionalTests, Error, TEXT("%s"), *Msg);
-			}
+			const FString Msg = FString::Printf(TEXT("%s [%s] %s : %s"), *WorkerName, (Require.bPassed ? *FString("Passed") : *FString("Failed")), *Require.Msg, *Require.StatusMsg);
```

Comment:
Are you sure you can't use string literals here? This requires more allocation etc.

Reply:
I've tried that and Chinese characters started appearing in the Unreal Test logs, using FString fixed this but I'm not sure if there are better ways of tackling this (e.g. would this leak memory?)

Reply:
possibly just TEXT("Passed"), TEXT("Failed")?

Reply:
This works, will fix it!